pecan/routing.py

```diff
@@ -170,18 +173,16 @@ def lookup_controller(obj, remainder, request=None):
 def handle_lookup_traversal(obj, args):
     try:
         result = obj(*args)
+    except TypeError as te:
+        logger.debug('Got exception calling lookup(): %s (%s)',
+                     te, te.args)
+    else:
```

Comment:
lol, first time I hear proper Python code is "out of scope".
I've moved it in a separate commit for the sake of it.